### PR TITLE
chore(CI): make relup in slim packages

### PR DESCRIPTION
**Pull request by terry-xiaoyu**

### CODE REVIEW

File: .github/workflows/build_slim_packages.yaml [hunk 1]
```diff
@@ -34,6 +34,28 @@ jobs:
         else
           echo "EMQX_NAME=emqx" >> $GITHUB_ENV
         fi
+    - name: downloads old emqx zip packages
```
[the remark this answers is not in the document]
Comment (zmstone) [reply .github/workflows/build_slim_packages.yaml:37]: So we can test it as a Makefile target

[the remark this answers is not in the document]
Comment (Rory-Z) [reply .github/workflows/build_slim_packages.yaml:37]: Do you want to use this step as a custom action in a workflow, or do you want to make it available to developers as a shell script?

[the remark this answers is not in the document]
Comment (zmstone) [reply .github/workflows/build_slim_packages.yaml:37]: available for developers to test locally in their dev machine.

[the remark this answers is not in the document]
Comment (Rory-Z) [reply .github/workflows/build_slim_packages.yaml:37]: Done for https://github.com/emqx/emqx/pull/7010

**Pull request closed without merging**